src/axom/slic/interface/slic.hpp

```diff
@@ -32,15 +32,29 @@ struct debug
 
 /*!
  * \brief Initializes the SLIC logging environment.
+ * \param [in] is_root Enables selective logging macros based on root.
```

Comment:
Suggestion: Should there be a note about the default behavior, i.e. that all ranks will consider themselves to be root.

And possibly the meaning of `is_root` for non-MPI configs/runs?
(the latter probably should go somewhere else (?)).